internal/server/feeds.go

```diff
@@ -21,15 +22,17 @@ type FeedsCache struct {
 	gc     gcache.Cache
 	ticker *time.Ticker
 	feeds  FeedsRepo
+	pp     *postprocessor.PostProcessor
 }
 
 // NewFeedsCache creates FeedsCache.
 // ttl - duration how often will be refresh cache for gets up-to-date feeds.
-func NewFeedsCache(feeds FeedsRepo, ttl time.Duration) *FeedsCache {
+func NewFeedsCache(feeds FeedsRepo, ttl time.Duration, pp *postprocessor.PostProcessor) *FeedsCache {
```

Comment:
NewFeedsCache receive FeedsRepo, and postprocessor.PostProcessor implement FeedsRepo, that's why you shouldn't pass pp to here.